### PR TITLE
[AzureMonitorExporter] cleanup Changelog before release

### PR DESCRIPTION
**Pull request by TimothyMothra**
Reviewing the Changelog before our next release.
I'm opening this PR so our team can discuss changes.

We already verified we have the correct sections:
- Features Added
- Breaking Changes
- Bugs Fixed
- Other Changes

### TODO
- review content
- review Azure SDK standards (https://github.com/Azure/azure-sdk/blob/main/docs/policies/releases.md#change-logs)

### CODE REVIEW

**Comment by TimothyMothra** [comment]
Azure SDK standards DOES specify how PR links should be formatted.
```
- Issue where AccountName on BlobUriBuilder would not be populated
  for non-IP style Uris ([#8638](https://github.com/Azure/azure-sdk-for-net/issues/8638))
```
I'll work on this change

**Comment by TimothyMothra** [comment]
I saw a planning email from Matt earlier today. Confirmed that the target release date is Friday Oct 7th.

File: sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md [hunk 1]
```diff
@@ -1,39 +1,39 @@
...
+  `Activity.Status` ([#31024](https://github.com/Azure/azure-sdk-for-net/pull/31024))
+* Changed `AzureMonitorTraceExporter` to internal ([#31067](https://github.com/Azure/azure-sdk-for-net/pull/31067))
+* Changed default offline storage directory from "Microsoft\ApplicationInsights" to "Microsoft\AzureMonitor" ([#31073](https://github.com/Azure/azure-sdk-for-net/pull/31073))
   Users may override the default location by setting `AzureMonitorExporterOptions.StorageDirectory`.
 
 ### Bugs Fixed
 
-* [Remove populating _MS.AggregationIntervalMs to all metrics](https://github.com/Azure/azure-sdk-for-net/pull/29473)
-* [Fix shared RoleName/RoleInstance between Trace and Log Exporter](https://github.com/Azure/azure-sdk-for-net/pull/26438)
+* Remove populating _MS.AggregationIntervalMs to all metrics ([#29473](https://github.com/Azure/azure-sdk-for-net/pull/29473))
```
Comment (vishweshbankwar) [inline sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md:31]:
```suggestion
```

File: sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md [hunk 2]
```diff
@@ -1,39 +1,39 @@
...
+* Fix shared RoleName/RoleInstance between Trace and Log Exporter ([#26438](https://github.com/Azure/azure-sdk-for-net/pull/26438))
 
 ### Other Changes
 
-* [Update persistent storage package dependency](https://github.com/Azure/azure-sdk-for-net/pull/29530)
+* Update persistent storage package dependency ([#29530](https://github.com/Azure/azure-sdk-for-net/pull/29530))
```
Comment (vishweshbankwar) [inline sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md:36]: Could be rephrased like "Update OpenTelemetry dependencies" with link to https://github.com/Azure/azure-sdk-for-net/pull/31065/files. This is the latest change, users will care about.

File: sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md [hunk 3]
```diff
@@ -1,39 +1,39 @@
...
 ### Breaking Changes
 
-* [Request and Dependency Success criteria will now be decided based on
-  `Activity.Status`](https://github.com/Azure/azure-sdk-for-net/pull/31024)
-* [Changed `AzureMonitorTraceExporter` to internal](https://github.com/Azure/azure-sdk-for-net/pull/31067)
-* [Changed default offline storage directory from "Microsoft\ApplicationInsights" to "Microsoft\AzureMonitor"](https://github.com/Azure/azure-sdk-for-net/pull/31073).
+* Request and Dependency Success criteria will now be decided based on
+  `Activity.Status` ([#31024](https://github.com/Azure/azure-sdk-for-net/pull/31024))
+* Changed `AzureMonitorTraceExporter` to internal ([#31067](https://github.com/Azure/azure-sdk-for-net/pull/31067))
+* Changed default offline storage directory from "Microsoft\ApplicationInsights" to "Microsoft\AzureMonitor" ([#31073](https://github.com/Azure/azure-sdk-for-net/pull/31073))
```
Comment (vishweshbankwar) [inline sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md:26]:
```suggestion
```

Comment (vishweshbankwar) [reply sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md:26]: This is not a breaking change as it was not released before.

File: sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md [hunk 4]
```diff
@@ -1,39 +1,39 @@
...
-* [Add exception telemetry](https://github.com/Azure/azure-sdk-for-net/pull/26670)
-* [Storage transmission evaluator](https://github.com/Azure/azure-sdk-for-net/pull/26976)
-* [Add metrics exporter](https://github.com/Azure/azure-sdk-for-net/pull/26651)
-* [Add default storage location and initialization for traces](https://github.com/Azure/azure-sdk-for-net/pull/26494)
-* [Change AzureMonitorExporterLoggingExtensions from internal to public](https://github.com/Azure/azure-sdk-for-net/pull/26355)
-* [Support for exporting Activity exception
-  event](https://github.com/Azure/azure-sdk-for-net/pull/29676)
-* [Update SampleRate in Request/Dependency telemetry](https://github.com/Azure/azure-sdk-for-net/pull/31118)
+* Add histogram metric type support ([#27544](https://github.com/Azure/azure-sdk-for-net/pull/27544))
+* Update OTel package dependencies to 1.2.0 ([#28507](https://github.com/Azure/azure-sdk-for-net/pull/28507))
```
Comment (vishweshbankwar) [inline sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md:8]:
```suggestion
```

File: sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md [hunk 5]
```diff
@@ -1,39 +1,39 @@
...
+* Add default storage initialization for logs and metrics ([#28506](https://github.com/Azure/azure-sdk-for-net/pull/28506))
+* Add disable storage switch ([#28446](https://github.com/Azure/azure-sdk-for-net/pull/28446))
+* Transmit from storage ([#26762](https://github.com/Azure/azure-sdk-for-net/pull/26762))
```
Comment (vishweshbankwar) [inline sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md:11]:
```suggestion
```

File: sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md [hunk 6]
```diff
@@ -1,39 +1,39 @@
...
+* Add exception telemetry ([#26670](https://github.com/Azure/azure-sdk-for-net/pull/26670))
```
Comment (vishweshbankwar) [inline sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md:12]:
```suggestion
* Added support for exception telemetry from ILogger ([#26670](https://github.com/Azure/azure-sdk-for-net/pull/26670))
```

File: sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md [hunk 7]
```diff
@@ -1,39 +1,39 @@
...
+* Storage transmission evaluator ([#26976](https://github.com/Azure/azure-sdk-for-net/pull/26976))
```
Comment (vishweshbankwar) [inline sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md:13]:
```suggestion
```

File: sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md [hunk 8]
```diff
@@ -1,39 +1,39 @@
...
+* Add metrics exporter ([#26651](https://github.com/Azure/azure-sdk-for-net/pull/26651))
+* Add default storage location and initialization for traces ([#26494](https://github.com/Azure/azure-sdk-for-net/pull/26494)
+* Change AzureMonitorExporterLoggingExtensions from internal to public ([#26355](https://github.com/Azure/azure-sdk-for-net/pull/26355))
+* Support for exporting Activity exception
+  event ([#29676](https://github.com/Azure/azure-sdk-for-net/pull/29676))
+* Update SampleRate in Request/Dependency telemetry ([#31118](https://github.com/Azure/azure-sdk-for-net/pull/31118))
```
Comment (vishweshbankwar) [inline sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md:19]:
```suggestion
* Added support for sampling using Application Insights based sampler.  ([#31118](https://github.com/Azure/azure-sdk-for-net/pull/31118))
```

File: sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md [hunk 9]
```diff
@@ -1,39 +1,39 @@
...
-* [Update SampleRate in Request/Dependency telemetry](https://github.com/Azure/azure-sdk-for-net/pull/31118)
+* Add histogram metric type support ([#27544](https://github.com/Azure/azure-sdk-for-net/pull/27544))
+* Update OTel package dependencies to 1.2.0 ([#28507](https://github.com/Azure/azure-sdk-for-net/pull/28507))
+* Add default storage initialization for logs and metrics ([#28506](https://github.com/Azure/azure-sdk-for-net/pull/28506))
+* Add disable storage switch ([#28446](https://github.com/Azure/azure-sdk-for-net/pull/28446))
+* Transmit from storage ([#26762](https://github.com/Azure/azure-sdk-for-net/pull/26762))
+* Add exception telemetry ([#26670](https://github.com/Azure/azure-sdk-for-net/pull/26670))
+* Storage transmission evaluator ([#26976](https://github.com/Azure/azure-sdk-for-net/pull/26976))
+* Add metrics exporter ([#26651](https://github.com/Azure/azure-sdk-for-net/pull/26651))
+* Add default storage location and initialization for traces ([#26494](https://github.com/Azure/azure-sdk-for-net/pull/26494)
```
Comment (vishweshbankwar) [inline sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md:15]:
```suggestion
```

File: sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md [hunk 10]
```diff
@@ -1,39 +1,39 @@
...
-* [Storage transmission evaluator](https://github.com/Azure/azure-sdk-for-net/pull/26976)
-* [Add metrics exporter](https://github.com/Azure/azure-sdk-for-net/pull/26651)
-* [Add default storage location and initialization for traces](https://github.com/Azure/azure-sdk-for-net/pull/26494)
-* [Change AzureMonitorExporterLoggingExtensions from internal to public](https://github.com/Azure/azure-sdk-for-net/pull/26355)
-* [Support for exporting Activity exception
-  event](https://github.com/Azure/azure-sdk-for-net/pull/29676)
-* [Update SampleRate in Request/Dependency telemetry](https://github.com/Azure/azure-sdk-for-net/pull/31118)
+* Add histogram metric type support ([#27544](https://github.com/Azure/azure-sdk-for-net/pull/27544))
+* Update OTel package dependencies to 1.2.0 ([#28507](https://github.com/Azure/azure-sdk-for-net/pull/28507))
+* Add default storage initialization for logs and metrics ([#28506](https://github.com/Azure/azure-sdk-for-net/pull/28506))
```
Comment (vishweshbankwar) [inline sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md:9]:
```suggestion
```

File: sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md [hunk 11]
```diff
@@ -1,39 +1,39 @@
...
-* [Transmit from storage](https://github.com/Azure/azure-sdk-for-net/pull/26762)
-* [Add exception telemetry](https://github.com/Azure/azure-sdk-for-net/pull/26670)
-* [Storage transmission evaluator](https://github.com/Azure/azure-sdk-for-net/pull/26976)
-* [Add metrics exporter](https://github.com/Azure/azure-sdk-for-net/pull/26651)
-* [Add default storage location and initialization for traces](https://github.com/Azure/azure-sdk-for-net/pull/26494)
-* [Change AzureMonitorExporterLoggingExtensions from internal to public](https://github.com/Azure/azure-sdk-for-net/pull/26355)
-* [Support for exporting Activity exception
-  event](https://github.com/Azure/azure-sdk-for-net/pull/29676)
-* [Update SampleRate in Request/Dependency telemetry](https://github.com/Azure/azure-sdk-for-net/pull/31118)
+* Add histogram metric type support ([#27544](https://github.com/Azure/azure-sdk-for-net/pull/27544))
```
Comment (vishweshbankwar) [inline sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md:7]:
```suggestion
```

File: sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md (hunk 4 above)
Comment (TimothyMothra) [reply sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md:8]: Do we need to mention 1.3.1 instead?

File: sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md (hunk 3 above)
Comment (TimothyMothra) [reply sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md]: Do we want a line item to mention that Persistent Storage is now enabled by default?

File: sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md [hunk 12]
```diff
@@ -1,39 +1,36 @@
...
+  - `AddAzureMonitorMetricExporter()` for Metrics ([#26651](https://github.com/Azure/azure-sdk-for-net/pull/26651))
+  - `AddAzureMonitorLogExporter()` for Logs ([#26355](https://github.com/Azure/azure-sdk-for-net/pull/26355))
+* Added support for offline storage when ingestion endpoint is unavailable. This is enabled by default.
+  - Default directory is "Microsoft\AzureMonitor" ([#31073](https://github.com/Azure/azure-sdk-for-net/pull/31073))
+  - Users may override the default location by setting `AzureMonitorExporterOptions.StorageDirectory` ([#26494](https://github.com/Azure/azure-sdk-for-net/pull/26494))
+  - Users may disable by setting `AzureMonitorExporterOptions.DisableOfflineStorage` ([#28446](https://github.com/Azure/azure-sdk-for-net/pull/28446))
+* Added support for exception telemetry from ILogger ([#26670](https://github.com/Azure/azure-sdk-for-net/pull/26670))
+* Support for exporting Activity exception event ([#29676](https://github.com/Azure/azure-sdk-for-net/pull/29676))
+* Added support for sampling using Application Insights based sampler.  ([#31118](https://github.com/Azure/azure-sdk-for-net/pull/31118))
```
Comment (vishweshbankwar) [inline sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md:17]: Does it make sense to include the following link here?
https://github.com/open-telemetry/opentelemetry-dotnet-contrib/tree/main/src/OpenTelemetry.Extensions.AzureMonitor#application-insights-sampler-for-opentelemetry-net

**Review by vishweshbankwar — APPROVED** [review]
LGTM - it looks clean and easy to understand now.

File: sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md (hunk 12 above)
Comment (TimothyMothra) [reply sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md:17]: Yes, I think so. I'll add that now

**Review by rajkumar-rangaraj — APPROVED** [review]
LGTM.

**Pull request merged**